src/pages/Search/SearchAdvancedFiltersPage/SearchFiltersCardPage.tsx

```diff
@@ -50,8 +51,12 @@ function SearchFiltersCardPage() {
         () => buildCardsData(workspaceCardFeeds ?? {}, userCardList ?? {}, personalDetails ?? {}, selectedCards, illustrations, true),
         [workspaceCardFeeds, userCardList, personalDetails, selectedCards, illustrations],
     );
+    const flattenedWorkspaceCardFeeds = Object.values(workspaceCardFeeds ?? {}).reduce<CardList>((result, domainCards) => {
+        Object.assign(result, domainCards);
+        return result;
+    }, {});
 
-    const domainFeedsData = useMemo(() => generateDomainFeedData(userCardList), [userCardList]);
+    const domainFeedsData = useMemo(() => generateDomainFeedData(flattenedWorkspaceCardFeeds), [flattenedWorkspaceCardFeeds]);
```

Comment:
Can you add a robust test for this logic to ensure we are covering the workspace and domain feeds well?